kinode/packages/chess/chess/src/lib.rs

```diff
@@ -8,27 +8,31 @@ use serde::{Deserialize, Serialize};
 use std::collections::{HashMap, HashSet};
 extern crate base64;
 
+use crate::kinode::process::chess_sys_api_v0::{
+    ChessRequest, ChessResponse, MoveRequest, NewGameRequest,
+};
+
 const ICON: &str = include_str!("icon");
 
 //
 // Our "chess protocol" request/response format. We'll always serialize these
 // to a byte vector and send them over IPC.
 //
 
-#[derive(Debug, Serialize, Deserialize)]
-enum ChessRequest {
-    NewGame { white: String, black: String },
-    Move { game_id: String, move_str: String },
-    Resign(String),
-}
-
-#[derive(Debug, Eq, PartialEq, Serialize, Deserialize)]
-enum ChessResponse {
-    NewGameAccepted,
-    NewGameRejected,
-    MoveAccepted,
-    MoveRejected,
-}
+//#[derive(Debug, Serialize, Deserialize)]
+//enum ChessRequest {
+//    NewGame { white: String, black: String },
+//    Move { game_id: String, move_str: String },
+//    Resign(String),
+//}
+//
+//#[derive(Debug, Eq, PartialEq, Serialize, Deserialize)]
+//enum ChessResponse {
+//    NewGameAccepted,
+//    NewGameRejected,
+//    MoveAccepted,
+//    MoveRejected,
+//}
```

Comment:
can delete this now that it's in wit, maybe add docstring to wit variant

Reply:
Done in 91717f